sources/packages/api/src-sql/StudentAccountApplication/Add-deleted-at-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.student_account_applications
+ADD
+    COLUMN IF NOT EXISTS deleted_at TIMESTAMP WITH TIME ZONE;
+
+COMMENT ON COLUMN sims.student_account_applications.deleted_at IS 'When set indicates that the record is considered deleted.';
```

Comment:
Comment can be slightly elaborate explaining that this column is for the purpose of type orm OOTB soft delete.